zerocopy-derive/src/output_tests.rs

```diff
@@ -197,19 +197,28 @@ fn test_try_from_bytes_enum() {
     test_derive! {
         TryFromBytes {
             #[repr(u8)]
-            enum ComplexWithGenerics<X, Y> {
+            enum ComplexWithGenerics<'a, const N: usize, X, Y: Deref>
```

Comment:
Just to also make sure that inline lifetime bounds are forwarded:
```suggestion
            enum ComplexWithGenerics<'a: 'static, const N: usize, X, Y: Deref>
```